coriolisclient/cli/replicas.py

```diff
@@ -248,6 +248,10 @@ def get_parser(self, prog_name):
                                  'networks on the destination.')
         parser.add_argument('--notes', dest='notes',
                             help='Notes about the replica.')
+        parser.add_argument('--force',
+                            help='Perform a forced update of a replica.',
```

Comment:
Please detail that "If set, errors on updating the Replica's parameters will be ignored. This may lead to an inconsistent Replica is misused."